docs/features/operators.md

```diff
@@ -0,0 +1,206 @@
+# Operators
+
+Operators are very useful for manipulating values, this will go though all of the operators currently in N:
+
+### `OR`
+The OR operator (`||` or `|`) can be used on two booleans or two ints, it will either output a bool or a int depending on its inputs.
```

Comment:
```suggestion
The OR operator (`||` or `|`) can be used on two booleans or two ints, it will either output a bool or an int depending on its inputs.
```